gcloud/gcloud_kms_integration_test.go

```diff
@@ -80,6 +83,13 @@ func (i *gcloudSecretTest) TestPutSecret(t *testing.T) error {
 	assert.NoError(t, err, "Unexpected error on PutSecret")
 	i.totalPuts++
 
+	// Put Secret with large secret data text
+	i.largePassphrase = randSeq(2000)
+	secretData[testSecretIdWithLargePassphrase] = i.largePassphrase
+	err = i.s.PutSecret(testSecretIdWithPassphrase, secretData, keyContext)
```

Comment:
Add a test for `GetSecret(testSecretIdWithLargePassphrase)` and ensure you get the same largePassphrase. Also lets add tests for the following cases:
- When the passphrase == rsa limit
- When the passphrase > rsa limit
- When the passphrase < rsa limit

A test for backward compatibility. You will need to Put the secret using the old method (w/o chunks) and ensure the new implementation of GetSecret still passes.

Reply:
updated :)